addons/l10n_cn/views/report_invoice.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="report_tax_invoice_document">
+            <t t-call="web.basic_layout">
+                <t t-set="o" t-value="o.with_context(lang=lang)" />
+                <div class="cn-tax-invoice">
+                    <div class="row">
+                        <div class="col-6 offset-3 p-3">
+                            <strong class="text-center cn-fs-25 cn-double-underline">Electronic Tax Invoice</strong>
+                        </div>
+                        <div class="col-3 p-3">
+                            <table class="table table-sm table-borderless">
```

Comment:
If i understand correctly, the voucher is already printed on the paper and should be filled (similar to how we do checks) so the whole layout should be appearing, only the data in the correct place.
Am I wrong?